src/tallies/tally.cpp

```diff
@@ -250,6 +254,42 @@ Tally::Tally(pugi::xml_node node)
     }
   }
 
+  // Currently the pulse-height tally is only working in a selected type of
+  // settings Here these settings are checked again
+  bool exists = std::find(std::begin(scores_), std::end(scores_), 1001) !=
+                std::end(scores_);
+  if (exists == 1) {
+    std::cout << "PULSE HEIGHT TALLY SIMULATION, VERSION 1.0, 15.09.2021"
+              << std::endl;
+  }
+  if (exists == 1 && scores_.size() > 1) {
+    std::cout << "Error: The Pulse-Height Tally can currently not be used in "
```

Comment:
OK, well, this somewhat resolves my previous comment about pulse heights. I don't think we should merge this if putting this limitation in, because this limitation is not due to any fundamental shortcoming of openmc but rather lack of time put into programming a design that can work with OpenMC's diverse, extensible tallying system.

Reply:
Yes that's part of the "tally problem". I need to pass the pht_storage to the tally quite differently, I'll try to tackle that in the next few days, think I need a bit of time for that.